pipelines/pingcap/tidb/latest/merged_integration_br_test.groovy

```diff
@@ -76,6 +76,12 @@ pipeline {
                     cache(path: "./bin", filter: '**/*', key: "ws/${BUILD_TAG}/br") {
                         sh label: 'tidb-server', script: 'ls bin/tidb-server || make'
                         sh label: 'build-for-br-integration-test', script: 'make build_for_br_integration_test'
+                        sh label: "download dependency", script: """
+                        chmod +x ../scripts/pingcap/br/*.sh
+                        ${WORKSPACE}/scripts/pingcap/br/integration_test_download_dependency.sh master master master master master http://fileserver.pingcap.net
```

Comment:
less readability, consider to define shell vars, or make the shell script better with named args.